devutils/src/devutils/invoke_utils.py

```diff
@@ -12,7 +12,7 @@
 
 from invoke import task
 
-ROOT = Path(__file__).parent.parent.parent
+ROOT = Path(__file__).resolve().parent.parent.parent
```

Comment:
Please don't `resolve()` here as it'll resolve symlinks which is not what we want.

Use `absolute()` instead.